monitoring/snmp/CEPH-PROMETHEUS-ALERT-MIB.txt

```diff
@@ -0,0 +1,377 @@
+CEPH-PROMETHEUS-ALERT-MIB DEFINITIONS ::= BEGIN
+
+IMPORTS
+    MODULE-IDENTITY, OBJECT-TYPE, NOTIFICATION-TYPE, enterprises, TimeTicks
+        FROM SNMPv2-SMI
+    DisplayString
+        FROM SNMPv2-TC
+;
+
+ceph       OBJECT IDENTIFIER ::= { enterprises 50495 }
+prometheus OBJECT IDENTIFIER ::= { ceph 15 }
```

Comment:
Have you run this through an MIB validator? I looked for an [online one](http://www.muonics.com/Tools/smicheck.php), and it's reporting some errors (maybe they're just false positives):

![image](https://user-images.githubusercontent.com/37327689/135504080-f71a7cc0-0514-40ed-ad89-66635d34cd83.png)

Reply:
I'll take a look. However, @votdev MIB is the best we have at this point. I'd like to get the MIB across and then build on it.

I know the MIB loads and I've been running tests with snmptrapd to verify the integration